themes-default/slim/views/vue-components/name-pattern.mako

```diff
@@ -0,0 +1,491 @@
+                    </div>
+                    <br>
+                </div>
+        
+                <!-- Anime only -->
+                <div v-if="animeType > 0" class="form-group">
+                    <label for="naming_anime" class="col-sm-2 control-label">
+                        <span>Add Absolute Number</span>
+                        <p>Add the absolute number to the season/episode format?</p>
+                    </label>
+                    <div class="col-sm-10 content">
+                        <input type="radio" name="naming_anime" id="naming_anime" value="1" v-model="animeType" @change="updatePatternSamples"/>
+                        <span>Only applies to animes. (eg. S15E45 - 310 vs S15E45)</span>
+                    </div>                    
+                </div>
+
+                <div v-if="animeType > 0" class="form-group">
+                    <label for="naming_anime_only" class="col-sm-2 control-label">
+                        <span>Only Absolute Number</span>
+                    </label>
+                    <div class="col-sm-10 content">
+                        <input type="radio" name="naming_anime" id="naming_anime_only" value="2" v-model="animeType" @change="updatePatternSamples"/>
+                        <span>Replace season/episode format with absolute number</span>
+                        <p>Only applies to animes.</p>
+                    </div>
+                </div>
+
+                <div v-if="animeType > 0"  class="form-group">
+                    <label for="naming_anime_none" class="col-sm-2 control-label">
+                        <span>No Absolute Number</span>
+                    </label>
+                    <div class="col-sm-10 content">
+                        <input type="radio" name="naming_anime" id="naming_anime_none" value="3" v-model="animeType" @change="updatePatternSamples"/>
+                        <span>Only applies to animes.</span>
+                        <p>Dont include the absolute number</p>
+                    </div>
+                </div>
+        </div>
+
+    </div>
+
+</script>
+
+
+<script>
+    Vue.component('name-pattern', {
+        template: '#name-pattern-tempate',
+        props: {
+            /**
+             * Current naming pattern.
+            */
+            namingPattern: {
+                type: String,
+                default: ''
+            },
+            /**
+             * An array of available preset naming patterns.
+            */
+            namingPresets: {
+                type: Array,
+                default: () => []
+            },
+            /**
+             * The selected multi ep style
+             */
+            multiEpStyle: {
+                type: Number,
+                default: null
+            },
+            /**
+             * Availale multi ep style
+             */
+            multiEpStyles: {
+                type: Array,
+                default: () => []
```

Comment:
No need for the function, just use `default: []`.

Reply:
All other fields using arrays are working fine. Does this get set something other than an array somewhere in the component?

Reply:
No. For example multiEpStyles is only assigned to this.availableMultiEpStyles on line 407

Reply:
@p0psicles marked as resolved. All good.